h2o-algos/src/main/java/hex/aggregator/Aggregator.java

```diff
@@ -193,6 +195,12 @@ public void computeImpl() {
             } else {
               Log.info(" Too few exemplars.");
               hi = mid;
+              if(previousNumExemplars == numExemplars)  count++;
```

Comment:
Does it mean the total number or "empty" iteration or only the number of the last "empty" iterations?

Is eg. a sequence like this possible?
no new examplar
no new exemplar
new examplars
no new examplar

Should the `count` be 3 (total) or just 1 (the last one after new examplars were found)

Reply:
the previousNumExemplars have the total of all exemplars found( in the above scenario it will be 3). So, if for `n` consecutive iterations, if the previousNumExemplars( which is the sum) remains the same, Aggregator needs to exit. Currently, it goes into an infinite loop.